doc/developers/contributing.rst

```diff
@@ -230,109 +234,456 @@ contribution must conform to the project's :ref:`coding guidelines
+            compiled extension being built with the clang and clang++ compilers installed by
+            conda with the ``-fopenmp`` command line flag in the log.
+
+    .. tab-item:: Linux
+      :class-label: tab-4
+
+      .. tab-set::
+        :class: tabs-package-manager
+
+        .. tab-item:: pip
+          :class-label: tab-6
+          :sync: package-manager-pip
+
+          To check your installed Python version, run:
+
+          .. prompt:: bash
+
+            python3 --version
+
+          If you don't have Python |PythonMinVersion| or later, please install `python3` from your
+          distribution's package manager.
+
+          Next, you need to have a working C/C++ compiler with OpenMP support.
+          Install the build dependencies for **Debian-based operating systems, e.g. Ubuntu**:
+
+          .. prompt:: bash $
+
+              sudo apt-get install build-essential python3-dev
+
+          Now create a virtual environment (venv_) and install the required python packages:
+
+          .. prompt:: bash
+
+            python -m venv sklearn-dev
+            source sklearn-dev/bin/activate  # activate
+            pip install wheel numpy scipy cython meson-python ninja \
+              pytest pytest-cov ruff==0.11.2 mypy numpydoc
+
+          .. dropdown:: If no isolated environment is used (not recommended!)
+
+            If, for some reason, you are not using an isolated environment (which is not recommended),
+            you need to use `pip3` instead of `pip`. You can check if it is installed with:
+
+            .. prompt:: bash
+
+              pip3 --version
+
+            and if not, install `python3-pip` from your distribution's package manager.
+
+            Next, install the required python packages with:
+
+            .. prompt:: bash
+
+              pip3 install wheel numpy scipy cython meson-python ninja
+
+            cython and the pre-compiled wheels for the runtime dependencies (numpy, scipy
+            and joblib) should then automatically be installed in
+            ``$HOME/.local/lib/pythonX.Y/site-packages``. In this case,
+            ``pip3`` also needs to be used instead of ``pip`` in the following steps and
+            when `installing scikit-learn <pip_build_>`_.
+
+
+          When precompiled wheels of the runtime dependencies are not available for your
+          architecture (e.g. **ARM**), you can install the system versions:
+
+          .. prompt:: bash $
+
+              sudo apt-get install cython3 python3-numpy python3-scipy
+
+          On **Red Hat and clones (e.g. CentOS)**, install the dependencies using:
+
+          .. prompt:: bash $
+
+              sudo yum -y install gcc gcc-c++ python3-devel numpy scipy
+
+        .. tab-item:: conda
+          :class-label: tab-6
+          :sync: package-manager-conda
+
+          To check your installed Python version, run:
+
+          .. prompt:: bash
+
+            python3 --version
```

Comment:
This should be removed I think, with `conda` the `conda create` command is going to install Python, so no need to check whether python is already installed

The sentence below should be adapted as well.